src/__tests__/strings-spec.js

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable */
+
+describe('Strings Library', () => {
+  describe('hash Method', () => {
+    const {hash} = require('../strings');
+
+    test('should return true if 32bit integer generated', () => {
+      let str = 'this is a sample string';
+      let result = hash(str);
+
+      expect(typeof str).toBe('string');
+      expect(typeof result).toBe('number');
+    });
+  });
+
+  describe('slugify Method', () => {
+    const {slugify} = require('../strings');
+
+    test('should return true when slugify method converts correctly', () => {
+      let str1 = 'THis Is A!   sample--string';
+      let str2 = 'this-is-a---sample--string';
+      let result = slugify(str1);
+
+      expect(result).not.toEqual(str1);
+      expect(result).toEqual(str2);
+    });
+  });
+
+  describe('String format Methods', () => {
+    const {capitalize, titleCase, camelCase, toUnderscoreCase} = require('../strings');
+
+    test('should return true when capitalize method converts correctly', () => {
+      let str1 = 'test test';
+      let str2 = 'Test Test';
```

Comment:
Then that's a bug. If you want Title case use the title case function. Capitalize should only capitalize the first letter.